src/electron/views/device-connect-view/components/device-connect-view-container.tsx

```diff
@@ -26,28 +31,30 @@ export type DeviceConnectViewContainerProps = {
     userConfigurationStoreData: UserConfigurationStoreData;
     deviceStoreData: DeviceStoreData;
     windowStateStoreData: WindowStateStoreData;
+    featureFlagStoreData: FeatureFlagStoreData;
+    androidSetupStoreData: AndroidSetupStoreData;
 };
 
 export const DeviceConnectViewContainer = NamedFC<DeviceConnectViewContainerProps>(
     'DeviceConnectViewContainer',
     props => {
         return (
             <div className={deviceConnectView}>
-                <WindowTitle
-                    pageTitle={'Connect to your Android device'}
-                    deps={props.deps}
-                    windowStateStoreData={props.windowStateStoreData}
-                >
-                    <HeaderIcon invertColors deps={props.deps} />
-                </WindowTitle>
                 <div className={mainContentWrapper}>
-                    <DeviceConnectBody
+                    <WindowTitle
```

Comment:
Was moving the WindowTitle inside the mainContentWrapper div intentional?

Reply:
nope, thanks for catching!